src/App.js

```diff
@@ -56,9 +49,6 @@ class App extends React.Component {
             <Route path="/profile">
               {this.state.user ? <Profile user={this.state.user} /> : <h3>No Profile Found </h3>}
             </Route>
-            <Route path="/create">
-              {this.state.showBookForm ? <BookFormModal /> : <AddBookButton onButtonClick={this.showBookFormHandler}/>}
-            </Route>
           </Switch>
           <Footer />
         </Router>
```

Comment:
Really nice work team! 💯 

Looks like you may need to add the Route path for home here :)